### ISSUE TITLE
Feature request: auto-reload config

### ISSUE THREAD

**Issue by nikitabobko** [nikitabobko/AeroSpace#163]
When the config changes AeroSpace can detect the file change and auto-reload the config

auto-reload behavior should be possible to disable in the config itself:
```
auto-reload-config = true|false
```

~~`reload-config` command should stop resetting to `main` mode?~~ (done)

Follow up is to make the value default: https://github.com/nikitabobko/AeroSpace/issues/1882

**Comment by KingSupernova31** [comment]
Agreed this would be great, rather frustrating trying to get Aerospace set up when I have to remember to reload after any change I make.

Looking at the linked PR, if listening for file changes is a pain to get working, couldn't aerospace just query the file every 100ms or so to see if it's changed? Slightly worse for performance, but I can't imagine that checking the "last updated" time of a file takes a significant amount of time.

**Comment by nikitabobko** [comment]
Fixed since 0.21.0-Beta
